app.py

```diff
@@ -1,120 +0,0 @@
-
-import os
-import praw
-
-from flask import Flask, redirect, request, Response
-
-
-app = Flask(__name__)
-Reddit = praw.Reddit(user_agent='slack-reddit-v1 by /u/arjunblj')
-
-SORT_TYPES = {
-    'hot': 'get_hot',
-    'new': 'get_new',
-    'top': 'get_top',
-    'rising': 'get_rising'
-}
-
-
-def _create_response(text):
-    """Returns a utf-8 encoded text response.
-    """
-    return Response(text, content_type='text/plain; charset=utf-8')
-
-
-def _parse_options(options):
-    """Returns a Response after validating the input.
-    """
-    if options[1] in SORT_TYPES.keys():
-        sort_type = options[1]
-        if len(options) > 2:
-            subreddit, sort_type, num_results = options
-            try:
-                num_results = int(num_results)
-                return fetch_posts(subreddit, sort_type, num_results)
-            except ValueError:
-                # Third option is invalid.
-                response_text = ("Sorry! `%s` isn't a valid sorting number. Try `/reddit %s %s 5`"
```

Comment:
Aren't all subreddits invalid sorting numbers? And what's the third option?

Also if what you're catching is the `int()` cast, you may want to put the return statement in the `else` clause. Makes it more clear where the exception could happen.